components/style/themes/default.less

```
@@ -532,10 +532,11 @@
 // --
 @progress-default-color: @processing-color;
 @progress-remaining-color: @background-color-base;
-@progress-text-color: @text-color;
+@progress-info-text-color: @text-color-secondary;
```

```suggestion
@progress-info-text-color: @progress-text-color;
```

否则修改 @progress-text-color 会失效。